app/exception/envelope_handlers.py

```
@@ -92,38 +92,51 @@ async def global_exception_handler_envelope(request: Request, exc: Exception):
+            message="서버 내부 오류가 발생했습니다. 담당자에게 문의해주세요.",
             code=ErrorCode.INTERNAL_ERROR,
-            result={
-                "error_detail": str(exc),
-                "stack_trace": traceback.format_exc()
-            } if IS_DEBUG else None
+            result=error_result
         ).model_dump()
     )
 
 
 async def rate_limit_exception_handler(request: Request, exc: RateLimitExceeded):
```

Ruff (0.15.0)

[warning] Unused function argument: `exc`

(ARG001)